resources/views/scheduleInstrument.html

```diff
@@ -129,78 +212,105 @@ <h3>Time Scheduled</h3>
+
+                    calendar = new FullCalendar.Calendar(calendarEl, {
+                        headerToolbar: {
+                            left: 'prev,next today',
+                            center: 'title',
+                            right: 'dayGridMonth,timeGridWeek,timeGridDay'
                         },
+                        initialDate: new Date(),
+                        navLinks: true, // can click day/week names to navigate views
+                        selectable: true,
+                        selectMirror: true,
+                        eventContent: function(e) {
+
+                            let content = '';
+                            let bgColor = (e.event.extendedProps.project == project ? e.backgroundColor : 'gray');
+                            let textColor = getContrastTextColor(stringToColor(bgColor));
+                            let dateStr = DateFormat.format.date(e.event.start, LABKEY.container.formats.dateTimeFormat) + ' - ' + DateFormat.format.date(e.event.end, LABKEY.container.formats.dateTimeFormat);
+                            let style = 'background-color: ' + bgColor + '; color: ' + textColor;
+                            content += '<div class="event-tooltip-content" style="' + style + ';">'
+                                        + '<div class="event-date">' + LABKEY.Utils.encodeHtml(dateStr) + '</div>'
+                                        + '<div class="event-id" >' + 'Project : &nbsp' + LABKEY.Utils.encodeHtml(e.event.extendedProps.project) + '</div>'
+                                    + '</div>';
+                            return { html: content };
+                        },
+                        eventMouseEnter: function (mouseEnterInfo) {
+                            removeEventLog();
+                            let instrumentID = instrument ? instrument : $('#instrumentDropDown select[name="instrumentDropDown"]').val();
+                            fetchInstrumentCosts(instrumentID, mouseEnterInfo.event.start, mouseEnterInfo.event.end, false);
+                            let content = '';
+                            let dateStr = DateFormat.format.date(mouseEnterInfo.event.start, LABKEY.container.formats.dateTimeFormat) + ' - ' + DateFormat.format.date(mouseEnterInfo.event.end, LABKEY.container.formats.dateTimeFormat);
+                            content += '<div class="event-tooltip-content">'
+                                    + '<div class="event-id" >' + 'Project Id : &nbsp' + LABKEY.Utils.encodeHtml(mouseEnterInfo.event.extendedProps.project) + '</div>'
+                                    + '<div class="event-title">' + 'Title : &nbsp' + LABKEY.Utils.encodeHtml(mouseEnterInfo.event.title) + '</div>'
+                                    + '<div class="event-date">' + LABKEY.Utils.encodeHtml(dateStr) + '</div>'
+
+                                    + '<div id="event-delete" role="button" style="cursor: pointer;">[Delete]</div>'
```

Comment:
Instead of introducing this as a styling approach, let's use `LABKEY.Utils.textLink()` here and other spots you're using brackets at the moment.